src/app/shared/http/http-responses.interceptor.ts

```diff
@@ -35,7 +35,9 @@ export class HttpResponsesInterceptor implements HttpInterceptor {
 	}
 
 	private handleHttpError(error: ObIObliqueHttpErrorResponse): Observable<never> {
-		return this.handleError(error, true, () => this.notify(error.error));
+		// 304 Not Modified is handled as an error status by Angular even though it's an OK status, so let's
+		// ignore the error.
+		return this.handleError(error, error.error.status !== 304, () => this.notify(error.error));
```

Comment:
What means "304" -> use a constant with a comprehensible name like HTTP-STATUS-NOT-MODIFIED